skimage/morphology/tests/test_rolling_ball.py

```diff
@@ -0,0 +1,39 @@
+"""
+Tests for Rolling Ball Filter
+(skimage.morphology.rolling_ball)
+
+Author: Sebastian Wallkötter
```

Comment:
Please, remove the author name. If you want to write it somewhere, use the file CONTRIBUTORS.txt at the root of the project.

Reply:
I used `test_selem.py` as a template, and they specified the author, so I thought it would be required.

It would be very nice if I could add my name to the CONTRIBUTORS; I'll do that once the PR is close to being completed and people are happy with the code.